dev/CommonStyles/TextBlock_themeresources_v2.5.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
We should keep the 2.5 styles the same as what we have today.

Reply:
Should we only add styles to that file than which don't exist in generic.xaml?

Reply:
Removed styles already defined in generic.xaml from this file, only introducing new styles, not overriding old ones.

Reply:
> Removed styles already defined in generic.xaml from this file, only introducing new styles, not overriding old ones.

That sounds like a good approach.